pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.test.ts

```diff
@@ -6,7 +6,7 @@ describe('component: LabeledInput', () => {
   it('should emit input only once', () => {
     const value = '2';
     const delay = 1;
-    const wrapper = mount(LabeledInput, {
+    const wrapper = mount(LabeledInput as any, {
```

Comment:
I'm not a fan of using `any` here, but I'm failing to find a solution that will satisfy vue test utils expectations here.

Reply:
It doesn't appear to cause any issues for me if you remove the type cast

![image](https://github.com/rancher/dashboard/assets/55104481/b61f378a-4f6a-4492-a986-1d5dfd9da3fd)

Reply:
It looks like the errors I'm receiving are reported by tsc. Can you let me know what the output of `yarn build:lib` is for you without the type cast?

Reply:
Yeah, running yarn build:lib fails. I've spent a heck of alot of time trying to make this work in a better way. I managed to get a variation of https://github.com/vuejs/vue-test-utils/issues/2026#issuecomment-1460894573 to partially work.

Unfortunately I had to pull in the entire test-utils definition in to fix other issues. I think using `any` is fine until we migrate to vue3 and a new version of test-utils.

Reply:
After posting this I did have one other thought... Why is the test getting built when running `yarn build:lib`? If it doesn't need to be bundled into the library I found this diff works fine

![image](https://github.com/rancher/dashboard/assets/55104481/dd0fff31-a02d-40f9-82b1-8c0c8bc21e51)

Reply:
Wouldn't that exclude all of our tests from the compilation process, which begs the question, what's the benefit of writing our tests in TypeScript in the first place?

I might opt for `any` for now and follow up in a separate PR to address the vue-test-utils issues. I'm seeing multiple github issues that appear to be related to the one we're experiencing here, https://github.com/vuejs/vue-test-utils/issues/2026 for example

Reply:
I think the tests are already being excluded from the compilation process since yarn test:ci functions as already noted. I personally don't see the point of typescript until composition or vue 3 is done.

I'm fine with you making the change later anyway. I just wanted to bring up what I found.